zerver/lib/actions.py

```diff
@@ -2989,3 +2997,21 @@ def make_dict(row):
     streams.sort(key=lambda elt: elt["name"])
 
     return streams
+
+def process_attachments(message):
+
+    def extract_path_id(upload_string):
+        cleaned_path = upload_string.replace('/user_uploads/', '')
+        cleaned_path = re.sub('[\(\)]', '', cleaned_path)
+        return cleaned_path
+
+    p = re.compile('\(.*?\)')
+    attachment_list = p.findall(message['message'].content)
```

Comment:
@timabbott There seems to be a bug in the code which renders the upload strings in messages to links.

` This is it ... [hello.txt](asd)` gets rendered to  `This is it ... hello.txt` with the hello.txt being rendered as a link.